### PR TITLE
Fix tests & update POJO

### PR DESCRIPTION
**Pull request by PavelEvleev**

### CODE REVIEW

File: api/src/main/java/app/services/AuthorService.java [hunk 1]
```diff
@@ -63,4 +47,20 @@ public void delete(Long id) {
...
         authorRepository.deleteAll();
     }
+
+    private AuthorInfo initAuthorInfo(Author author) {
+        return new AuthorInfo() {{
+            setName(author.getName());
+            setId(author.getId());
+            if (author.getBooks() != null)
+                setBooks(author.getBooks().stream().map((i) -> new BookInfo() {{
+                    setId(i.getId());
```
Comment (lehaSVV2009) [inline api/src/main/java/app/services/AuthorService.java]: really I've never seen something like new `MySimpleObject {{ setX().. setY() }}` Not sure if it is good or bad. But not usual :-)

Comment (lehaSVV2009) [inline api/src/main/java/app/services/AuthorService.java]: Try not to create variables with one character like `i` or `n`. It smells bad

File: api/src/main/java/app/services/BookService.java [hunk 2]
```diff
@@ -46,18 +46,18 @@ private BookInfo initBookInfo(Book book) {
...
         List<Long> authorsId = book.getAuthorsIds();
-        List<Author> authors = authorsId.stream()
-                .map(id -> authorService.findOneEntity(id)).collect(Collectors.toList());
-        Book newBook = new Book(book.name,
-                book.publisher, Date.valueOf("2017-03-01"));
-        newBook.setAuthors(authors);
+        Book newBook = new Book(book.name, book.publisher, Date.valueOf("2017-03-01"));
+        if (authorsId != null) {
+            List<Author> authors = authorsId.stream()
+                    .map(id -> authorService.findOneEntity(id)).collect(Collectors.toList());
```
Comment (lehaSVV2009) [inline api/src/main/java/app/services/BookService.java]: I would better use repository for it.. findOneEntity looks worse

File: api/src/main/java/app/services/AuthorService.java (hunk 1 above)
Comment (PavelEvleev) [reply api/src/main/java/app/services/AuthorService.java]: Instead **`i`** insert **`bookItem`**?

**Pull request merged**